src/screens/DataTable.js

```diff
@@ -275,14 +272,30 @@ export default () => (
               If object is specified 'property' is used to group data by,
               'expand' accepts array of group keys that sets expanded groups and
               'onExpand' is a function that will be called after expand button
-              is clicked with an array of keys of expanded groups.
+              is clicked with an array of keys of expanded groups. If 'onUpdate'
+              is being used to lazy load group items, 'expandable' can be used
+              to specify which items in the data are group header items and
+              'select' and 'onSelect' can be specified in 'groupBy' to help
+              specify the selection state of the groups and handle changes in
+              group selection state.
             </Description>
             <Example>
               {`
 {
   "property": "location",
   "expand": ["Paris", "Los Angeles"],
   "onExpand": "(key) => {...}"
+}
+            `}
+            </Example>
+            <Example>
+              {`
+{
+  "property": "location",
+  "expand": ["Paris", "Los Angeles"],
+  "expandable": ["Paris", "Los Angeles", "Fort Collins", "San Jose"],
+  "select": { "": "some", "Paris": "all", "Los Angeles": "some"}
```

Comment:
What does empty string refer to, the entire table? was this chosen to avoid any potential conflicts with user data?

wondering if we can be more clear about this in the description above about how to use `select` in this way

Reply:
Yes and yes. This is something we didn't think about and I ran into during implementation. We can probably talk about this more and come up with some better descriptions if nothing else.